index.js

```diff
@@ -1,34 +1,53 @@
-import makeTemplate from "lodash.template";
-import { visitParents as visit } from "unist-util-visit-parents";
+/**
+ * A rehype plugin to replace template strings based on lodash.template.
+ *
+ * @since 0.0.0
+ * @package rehype-lodash-template
+ * @author Viktor Yakubiv (www.yakubiv.com)
+ * @copyright Viktor Yakubiv & Contributors
+ * @license MIT
+ */
 
-const attach = (options = {}) => {
-  const { values = {}, templateSettings } = options;
+import makeTemplate from "lodash.template";
+import { visitParents } from "unist-util-visit-parents";
 
-  const cache = new Map();
+/**
+ * A rehype plugin to replace template strings based on lodash.template.
+ *
+ * @since 0.0.0
```

Comment:
There was no version 0.0.0, you can look it up via `git tag` – the earliest was `0.0.1`